src/routes/game/components/GameOverlays.vue

```diff
@@ -15,7 +15,7 @@
         :loading="leavingGame"
         @click="goHome"
       >
-        Leave Game
+        {{ t('game.overlays.leaveGame') }}
```

Comment:
I thought about combining this and 'waitingForGameToStart` into one block, but they're the only overlay that has more than one piece of text so I think it isn't worth grouping them.